src/test/kotlin/com/kafkakotlin/demo/kafka/producer/ProducerTest.kt

```diff
@@ -0,0 +1,21 @@
+package com.kafkakotlin.demo.kafka.producer
+
+// import com.kafkakotlin.demo.kafka.KafkaConfiguration
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.kafka.test.context.EmbeddedKafka
+import org.springframework.test.context.TestPropertySource
+import org.springframework.test.context.junit.jupiter.SpringExtension
+
+@SpringBootTest()
+@TestPropertySource(properties = ["/application-test.properties"])
+@EmbeddedKafka(topics = ["user-topic"])
+@ExtendWith(SpringExtension::class)
+class ProducerTest {
```

Comment:
In CMS this test is more to test the Kafka configuration of the entire app, it isn't really anything to do with Producers specifically